backend/build.gradle

```diff
@@ -194,6 +194,20 @@ spotless {
     }
 }
 
+task testDbStart(type: Exec) {
+    environment "SR_DB_PORT", testDbPort
+    commandLine "docker-compose", "-p", "simple-report-tests", "-f", "../docker-compose.yml", "up", "-d", "db"
```

Comment:
this needs to be `docker compose` (with space)

Reply:
Good call! I'll try to fix other spots that still have the dash too